### PR TITLE
[DO NOT REVIEW] Prototype RAR as service using out of procese infrastructure

### PR DESCRIPTION
**Pull request by rokonec**
Prototype! Used only for internal code discussions. Will be thrown away eventually.

### CODE REVIEW

File: src/MSBuildTaskHost/MSBuildTaskHost.csproj [hunk 1]
```diff
@@ -45,6 +45,9 @@
     <Compile Include="..\Framework\BuildEngineResult.cs">
       <Link>BuildEngineResult.cs</Link>
     </Compile>
+    <Compile Include="..\Framework\TaskExecutionContext.cs">
+      <Link>BuildEngineResult.cs</Link>
```
Comment (AR-May) [inline src/MSBuildTaskHost/MSBuildTaskHost.csproj:49]: Wrong link?

File: src/MSBuild/MSBuild.csproj [hunk 2]
```diff
@@ -187,6 +187,8 @@
     </Compile>
     <Compile Include="NodeEndpointOutOfProcTaskHost.cs" />
     <Compile Include="LogMessagePacket.cs" />
+    <Compile Include="NodeEndpointRarTaskHost.cs" />
+    <Compile Include="RarServiceTaskHostNode .cs" />
```
Comment (AR-May) [inline src/MSBuild/MSBuild.csproj:191]: Extra space.

Comment (rokonec) [reply src/MSBuild/MSBuild.csproj:191]: interesting... i haven't modified iot by hand + how come it compiles?

Comment (rokonec) [reply src/MSBuild/MSBuild.csproj:191]: fixed anyway

File: src/MSBuildTaskHost/MSBuildTaskHost.csproj (hunk 1 above)
Comment (rokonec) [reply src/MSBuildTaskHost/MSBuildTaskHost.csproj:49]: link fixed, still how come it compiles...?

File: src/MSBuild/MSBuild.csproj (hunk 2 above)
Comment (AR-May) [reply src/MSBuild/MSBuild.csproj:191]: >how come it compiles?

that's the question I asked myself too..

**Pull request closed without merging**